src/explore-education-statistics-common/src/services/releaseMetaGuidanceService.ts

```diff
@@ -15,6 +15,10 @@ export interface SubjectMetaGuidance {
     label: string;
     value: string;
   }[];
+  footnotes: {
```

Comment:
I know we've got various id / text pair types stashed around that you might or might not wanna make use of e.g. `IdLabelPair`.

Reply:
Probably gonna keep it as is in case we need to change it in the future. Tbh, `IdLabelPair` is such a simple type that it's not really gaining us anything by using it.